### ISSUE TITLE
Latest update broke everything on the xbox

### ISSUE THREAD

**Issue by AncientMystic** [jellyfin/jellyfin-uwp#39]
After the latest update on the xbox broke basically everything. 

the keyboard no longer works to control the ui

it crashes a lot, its hard to get it to go to the main default media selection page without crashing 

requires input of the server address at every startup

doesn’t playback properly

freezes every 3 min during playback

my media server somehow started doing the freezing every three minutes on every device after this update, its been great for months until this update and it just has been horrible ever since, now needs restarting constantly to get it to play i am going to have to restore it from a backup or something

it constantly jumps back and replays the same scenes from media over and over again when you get it to start playing again after freezing even if you skip ahead it usually tries to jump back to where it tried to start off again

it won’t respond to remote control from the webui on a browser

when playback is finally achieved for the brief 3 min period the cursor wont disappear unless you unplug/turn off the controller. 

xbox has no option to downgrade to previous version so when this update is pushed you’re stuck with it and the only option is to uninstall and not use jellyfin and this update is completely unusable. 

I personally also don’t like the new edge chromium base i prefer the old one 

i have it installed on the xbox one X model if the specific xbox model matters

**Comment by Killerherts** [comment]
A roll back is already in progress I do not think it has been released yet as my app has not updated again this is being track on issue #36

**Comment by AncientMystic** [comment]
Awesome, thank you for letting me know and ill check that issue out 👍🥳

**Comment by AncientMystic** [comment]
> A roll back is already in progress I do not think it has been released yet as my app has not updated again this is being track on issue #36

I just got the old one again it doesn’t show as an update since its a lower version number i had to uninstall and reinstall to get the rollback. So just in case you or anyone else is waiting on it gotta reinstall it 👍😁

**Comment by AncientMystic** [comment]
I am now getting the error 

“We’re unable to connect to the selected server right now. Please ensure it is running and try again” 

when using the lan address and default port that has always worked before and is still working on every other device on the network

**Comment by baddestguy** [comment]
Getting the same error as @AncientMystic . Did you find a fix?

**Comment by AncientMystic** [comment]
> Getting the same error as @AncientMystic . Did you find a fix?

I have not, my fix so far was since i do not want to use the built in edge browser ( which so far seems like the suggested alternative but i hate using anything chromium based but thats another story) and i have a proxmox server up and running i used a virtual machine and the sunshine/moonlight software for streaming games to just connect into the virtual machine and watch jellyfin over moonlight and will likely just be replacing the xbox in time since i was planning on doing that anyways and was only using it as a media device because i received someones old one x for free

**Comment by Killerherts** [comment]
This won't be fixed until my pr is merged, and it's published, sorry about that guys. 
edge and kodi are only options on console right now

**Comment by AncientMystic** [comment]
> This won't be fixed until my pr is merged, and it's published, sorry about that guys. edge and kodi are only options on console right now

No worries, thank you for all your hard work providing this option to us

**Comment by Killerherts** [comment]
When #41 is published these issues should be resolved the fixes have been merged to the master we are just waiting on publishing from Microsoft. 

@anthonylavado do you need anything else ?

**Comment by dfederm** [comment]
@anthonylavado just checking to see what the status is for publishing

**Comment by anthonylavado** [comment]
Hi! Just got back from vacation. Thanks for the heads up, will open up VS and send off tonight.

**Comment by AncientMystic** [comment]
Just tried it again, i keep checking but still not working, as it is i just get an error and it will not allow you to add a server to use the app at all.

Would it be possible to just completely revert to the old version **exactly** how it was before any changes at all were made? 

It may have had a few little issues here and there but at least it worked

**Comment by anthonylavado** [comment]
@AncientMystic I just sent a version out for certification. Sorry, there are so many issues open for similar topics, which meant that I didn't update this one. I'll try to link some together to help centralize things.

**Comment by AncientMystic** [comment]
> @AncientMystic I just sent a version out for certification. Sorry, there are so many issues open for similar topics, which meant that I didn't update this one. I'll try to link some together to help centralize things.

Totally understandable and thank you, it is very much so appreciated, thank you for all your hard work jellyfin is one of my favourite softwares

**Comment by AncientMystic** [comment]
Just reinstalled the app and tested it, it is now working again as expected 👍 

thank you again for your hard work on this project.

**Comment by anthonylavado** [comment]
Yeah, looks like everything is "updated" again, and things are working so far in my basic tests. Now on to improvements...

**Comment by Kreuger** [comment]
I havent seen an update on my xbox and it is still constantly asking for my server IP.

**Comment by Killerherts** [comment]
> I havent seen an update on my xbox and it is still constantly asking for my server IP.

You have to uninstall and reinstall we didn't change the version number
